cupy/indexing/generate.py

```diff
@@ -252,7 +252,57 @@ def ix_(*args):
 # TODO(okuta): Implement ravel_multi_index
 
 
-# TODO(okuta): Implement unravel_index
+def unravel_index(indices, dims, order='C'):
+    """Converts a flat index or array of flat indices into a tuple of coordinate arrays.
+
+    Args:
+        indices: An integer array whose elements are indices
+            into the flattened version of an array of dimensions :obj:`dims`.
+        dims: The shape of the array to use for unraveling indices.
+        order: Determines whether the indices should be viewed as indexing
+            in row-major (C-style) or column-major (Fortran-style) order.
+
+    Returns: tuple of ndarrays:
+        Each array in the tuple has the same shape as the indices array.
+
+    Examples
+    --------
+    >>> cupy.unravel_index(cupy.array([22, 41, 37]), (7, 6))
+    (array([3, 6, 6]), array([4, 5, 1]))
+    >>> cupy.unravel_index(cupy.array([31, 41, 13]), (7, 6), order='F')
+    (array([3, 6, 6]), array([4, 5, 1]))
+
+    .. seealso:: :func:`numpy.unravel_index`
+
+    """
+    if order == 'C':
+        dims = reversed(dims)
+    elif order == 'F':
+        pass
+    else:
+        raise TypeError("order not understood")
```

Comment:
Please use single quotes. (Applies for other string literals as well).

Reply:
I also prefer single quotes. However, the docs of other functions use double quotes. So I followed them.

Reply:
We use triple double quotes in docstrings and use single quotes for normal string literals.

Reply:
https://github.com/cupy/cupy/blob/master/cupy/indexing/generate.py#L242

Reply:
That's against the style of other part of code and something we should fix.

Reply:
OK. I think all rules which you pointed are not documented in https://docs.chainer.org/en/stable/contribution.html